core/templates/dev/head/components/explorationSummaryTile.js

```diff
@@ -42,7 +42,7 @@ oppia.directive('explorationSummaryTile', [function() {
       function($scope, oppiaDatetimeFormatter, ratingComputationService) {
         $scope.getLastUpdatedDatetime = function() {
```

Comment:
I think I'd rather leave it for now, since we'll probably use it later for computing recency. What do you think?